hal/hal_uv_client_usb.c

```diff
@@ -27,6 +27,9 @@ typedef struct adb_client_usb_s {
     adb_client_uv_t uc;
     /* FIXME libuv handle must be right after adb_client_uv_t */
     uv_pipe_t pipe;
+#ifdef CONFIG_ADBD_USB_TWOPIPE
```

Comment:
@spiriou do we need support the old protocol if the user could switch to the old protocol by git revert or git checkout?

Reply:
I was also thinking the same thing. This `hal/hal_uv_client_usb.c` is only used by NuttX so it makes sense to align the protocol without `#ifdef`.

It would also be more explicit to rename the pipes `ep_in` and `ep_out`, instead of `pipe` and `pipe_2`.

Reply:
done